approvals.go

```diff
@@ -302,3 +309,47 @@ func alwaysOption(opts []verifyOptions) verifyOptions {
 
 	return v
 }
+
+// AcceptChanges handles renaming `.received` files to `.approved` based on the given flag.
+// If `approve` is true, it renames all `.received.txt` and `.received.json` files in the default folder
+// to `.approved.txt` and `.approved.json`, respectively.
+func AcceptChanges(approve bool) error {
+	if !approve {
+		return nil
+	}
+
+	if defaultFolder == "" {
+		return fmt.Errorf("default folder is not set; use UseFolder to set the folder")
+	}
+
+	return filepath.Walk(defaultFolder, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		// Skip directories
+		if info.IsDir() {
+			return nil
+		}
+
+		// Handle `.received.txt` files
```

Comment:
might just want to look for files ending in `.received.<ext>` or... is blank `.received` also valid? since it's a library